congress_lookup.py

```diff
@@ -61,8 +61,8 @@ def inclusive_wildcard_match(self,name,pat):
     def lookup_by_member(self,property,member):
         for leg in ( leg for leg in self.legislators if \
                     (leg['name']['official_full'] == member['name']) \
-                    or ('bioguide' in leg['id'] and leg['id']['bioguide'] == member['bioguide']) \
-                    or ('thomas' in leg['id'] and leg['id']['thomas'] == member['thomas']) ):
+                    or ('bioguide' in leg['id'] and 'bioguide' in member and leg['id']['bioguide'] == member['bioguide']) \
+                    or ('thomas' in leg['id'] and 'thomas' in member and leg['id']['thomas'] == member['thomas']) ):
```

Comment:
So you know, you can do `dict.get('key')` instead of `dict['key']`, and the `.get()` version will just return `None` by default if the key doesn't exist, rather than throwing an exception.

You can also `dict.get('key', 'default value')` and have it return `'default value'` (which can be anything) if the key doesn't exist, rather than `None`.